include/revng/Support/AccessTracker.h

```diff
@@ -17,14 +17,18 @@ namespace revng {
 class AccessTracker {
```

Comment:
AFAIU upon construction we're not tracking, when we call `.clear` we start tracking.
This is quite unexpected for a class called `AccessTracker`.
What's the ratio here?

Reply:
we went for this solution because it class is embedded everywhere in the model, so by default we do not wish to track, and when starting to track we always want to clear, and when we clear we always want to track, so i saw no point in exposing two primitives. I can't really find a better name than this, except maybe `clearAndStartTracking`